chimney/.jvm/src/test/scala/io/scalaland/chimney/WixSpec.scala

```diff
@@ -264,13 +264,29 @@ object WixSpec extends TestSuite {
         EntityDTO(None).into[Entity].transform ==> Entity(SdlMissingIdPlaceholderString)
       }
 
+      "use provided value if id is Some (IdGeneration.Auto)" - {
```

Comment:
should you also do it for UUID?

Reply:
I think no, the underlying code is being reused. I'm essentially testing, that macros uses `.getOrElse`, and not disregards the Some value (whatever it might be).

Reply:
👍